### ISSUE TITLE
boot_timeout

### ISSUE THREAD

**Issue by JaminShanti** [gusztavvargadr/packer#149]
Hey thanks for making these, very helpful.
I am using windows-10 v1903.0.1909 and it's timing out on boot up.
What do you suggest for the config.vm.boot_timeout ?

**Comment by gusztavvargadr** [comment]
Hi @JaminShanti, thanks, glad to hear they are useful (especially if they boot up :) The default timeout of the boxes should be 10 minutes which is usually enough for the scenarios I test with.

By default the boxes are configured with 1 vCPU and 1GB of RAM (so VMs can be created on most hosts), but this might be a bit low for the latest Windows 10, especially for the first boot as boots after Sysprep. You might want to try with at least 2 vCPUs and 2GB or 4GB of RAM. Please share some details of your host (OS and virtualization provider versions, CPU and RAM and utilization), so I can try to reproduce.

**Comment by JaminShanti** [comment]
Hello, Thanks for your reply.

Here's my setup, https://github.com/JaminShanti/ansible_lab.

Currently, I am working towards being able to run win_ping from the ansible/centos instance to lab1/win10 instance.  I know I have more work to do,  Presently getting an idea when the win10 instance is available and that it gets the IP I assign.

**Comment by JaminShanti** [comment]
noticing it's not running, 
lab1.vm.network :private_network, ip: '192.168.50.101'

**Comment by gusztavvargadr** [comment]
@JaminShanti I'm not sure I understand the problem exactly, can you try to clarify?

I have tried the setup you provided and the Windows 10 machine boots with no problem in about 5 mins on my host. If you want the other machine to manage it remotely, I'd suggest to swap their order in the Vagrantfile, that is, boot the Windows 10 machine first, then the CentOS one - Vagrant should take care of starting the second machine only after the first one is up and available.

For more complex scenarios you might want to take a look at Vagrant triggers: https://www.vagrantup.com/docs/triggers/

Also, I recommend the plugins at https://github.com/hashicorp/vagrant/wiki/Available-Vagrant-Plugins#local-domain-resolution for easier management of networking, once you got the core setup working.

**Comment by JaminShanti** [comment]
I took your suggestions.  getting lab1: WinRM transport: negotiate timeouts sometimes. I'll post again with some more tests.

**Comment by gusztavvargadr** [comment]
@JaminShanti yes please, it would help if you could provide full Vagrant logs of your commands and maybe the details of your host environment (OS and Vagrant version).

**Comment by JaminShanti** [comment]
Got it to work with some new updates, they are checked in.
I had to extend the timeout to 1200.

The vagrant dns plugins were nice, maybe later.

ansible: lab1 | SUCCESS => {                                                                                                                                    ansible:     "changed": false,                                                                                                                                  ansible:     "ping": "pong"                                                                                                                                     ansible: }

**Comment by gusztavvargadr** [comment]
@JaminShanti glad to hear it worked and that you find the boxes useful :)

Indeed, first boots are CPU intensive, you might want to consider production scenarios on something a bit more powerful (I build and test usually with recent generation Core i7, so our measurements obviously vary :)

Please let me know what is your use case in more detail. If you only need the basic Windows environment, you can consider the Windows Server boxes, especially the Core (that has no UI) boots significantly faster, Windows 10 is optimized for the desktop experience. https://app.vagrantup.com/gusztavvargadr/boxes/windows-server

I'm closing this now, but feel free to open a new ticket on new box suggestions, I'm glad to extend these if there's need (that's the whole point if this "experiment" on my end :)

**Comment by JaminShanti** [comment]
Thanks for your help on this.  I am going to try the standard-core image for faster boot time.  My project is on windows10, though I agree that's a strange configuration.  I have been looking for good windows vagrants forever.  Thanks for creating these.
